pbl

```diff
@@ -0,0 +1,224 @@
+#! /usr/bin/perl
```

Comment:
For the moment it does do. The rest is kept for compat as well. But The script can be rewritten in bash later.